### ISSUE TITLE
Request - Add custom field for location

### ISSUE THREAD

**Issue by KoolPal** [iamaminemesbah/manual-reviews-for-woocommerce#2]
Hi,

I am glad to see you back (I am assuming you are the original author from https://wordpress.org/plugins/manual-reviews-for-woocommerce/)

Would it be possible to add a custom field Location and make the editable as well as display on the front end?

Alternatively would be possible to allow users of the plugin to define custom fields and then use it as they deem fit?

Thanks

**Comment by iamaminemesbah** [comment]
Yes that’s me. The plugin was deleted from Wordpress directory so i made it available on github.
I will look into this and I’ll see what i can do.
